packages/@uppy/core/src/__snapshots__/index.test.js.snap

```diff
@@ -27,14 +27,14 @@ Object {
       "name": "foo.jpg",
       "preview": undefined,
       "progress": Object {
-        "bytesTotal": 0,
+        "bytesTotal": null,
         "bytesUploaded": 0,
         "percentage": 0,
```

Comment:
Should all those be `null` then? Otherwise `bytesTotal` is `null`, but `bytesUploaded` is `0`?

Reply:
I only changed `bytesTotal` because `bytesUploaded: 0` is actually accurate, we did upload precisely 0 bytes, while `bytesTotal: 0` was a lie (we don't know if the file is 0 bytes)